### PR TITLE
Works-About_Us-Form-Sections

### PR DESCRIPTION
**Pull request by miltonHenschel**
- Added works sections
- Added about us sections
- Added form section
- Added social media buttons at bottom

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,5 +25,188 @@ <h2>I’am a Software Developer.</h2>
...
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a standard
+            dummy text. A daily selection of privately personalized reads; no
+            accounts or sign-ups required.
+          </p>
+          <div class="lang-box">
+            <span>css</span>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly put all the project technologies (e.g., "JavasScript", "HTML" and "CSS") into a list  of `ul` and each of them into a `li`

File: index.html [hunk 2]
```diff
@@ -23,5 +25,188 @@ <h2>I’am a Software Developer.</h2>
...
+            <a href=""></a><button class="flex-button">See Project</button></a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+    <footer class="footer">
+      <h1>About me</h1>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly follow the proper requirements for styling the project

Screen Shot 2022-10-29 at 05 24 54

File: index.html [hunk 3]
```diff
@@ -23,5 +25,188 @@ <h2>I’am a Software Developer.</h2>
...
+            <span>bootstrap</span>
+            <span>ruby</span>
+          </div>
+          <a href=""></a><button>See Project</button></a>
+        </div>
+        <div class="demo-wrap">
+          <div class="main-projects  demo-content">
+            <h3>Professional Art<br />Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
```
Comment (Kossi-stack) [inline index.html:61]: - [ ]  Kindly follow the styling  requirements for the project
Screen Shot 2022-10-29 at 05 24 07

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @miltonHenschel,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (miltonHenschel) [reply index.html]: Done. Thanks.

File: index.html (hunk 3 above)
Comment (miltonHenschel) [reply index.html:61]: Thanks.

File: index.html (hunk 2 above)
Comment (miltonHenschel) [reply index.html]: Thanks.

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 

Hi @miltonHenschel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

bunnys

## Highlights 💯 
- No linter errors ✔️ 
- Descriptive Pull request ✔️ 
- Good commit messages ✔️ 
- Good use of Github flow ✔️ 
- Professional Readme file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by miltonHenschel** [comment]
Finally! Thanks very much for the review.

**Pull request merged**